cosi.proto

```diff
@@ -126,7 +170,7 @@ message ProvisionerDeleteBucketRequest {
     
```

Comment:
`string bucket_name` should not be an input anywhere other than `CreateBucket`, the `CreateBucket` call should return a `string bucket_id` (generated by the plugin), and all subsequent calls should only operate on `string bucket_id` NOT `string bucket_name`.

Reply:
bucket_name is already the unique id